smdebug/tensorflow/keras.py

```diff
@@ -616,10 +618,14 @@ def _save_layer_input_and_outputs(self):
             )
             t = tensor[0] if isinstance(tensor, list) and len(tensor) else tensor
             if hasattr(t, "numpy") is False:
-                self.logger.warning("cannot save layer values during forward pass with tf.function")
+                contain_layer_value = True
+                continue
             else:
                 self._save_tensor_to_file(export_name, tensor, output_collection)
 
+        if contain_layer_value:
```

Comment:
This solution mitigates the problem but doesn't really solve the issue. See this example job I ran after building `smdebug` off of this branch: https://tiny.amazon.com/dun1clc1/IsenLink.

There are three possible solutions from here:
1. Remove the logging statement completely, and we can revisit a better way to log the message **(this is my recommendation)**
2. Log this message once, and keep track of a flag to make sure it doesn't get logged again (similar to the profiler config parser)
3. Log the tensor + relevant information with the message so that it isn't duplicate information in each logging message.